src/Zadatak2/Rectangle.java

```diff
@@ -0,0 +1,46 @@
+package Zadatak2;
```

Comment:
I didn't explain this. Package names follow the naming convention: {domain}.{organisation}.{project}.{...}
i.e. org.kanita-sa.qamp.zadaci.zadaca5

Reply:
okay, I fixed it. I hope it is correct. The only question that I have is what's the difference? Of course, besides the obvious. Since the code is working with the previous version, and the current version. What does the change I made changes in our project?